src/renderers/common/RenderObject.js

```diff
@@ -317,6 +317,12 @@ export default class RenderObject {
 
 		}
 
+		if ( object.center ) {
+
+			cacheKey += object.center.x + ',' + object.center.y + ',';
```

Comment:
uniform values ​​should not generate cache key.

Reply:
Unfortunately center is not a uniform value.

In that case we probably want to deprecate that property and move it to the SpriteNodeMaterial as SpriteNodeMaterial.centerNode?

Reply:
It seems to be uniform in both renderers, but I think you may have a problem related to the uniform reference. Maybe if you replace `uniform()` with `reference( 'center', 'vec2' )` can it work in your tests?

https://github.com/mrdoob/three.js/blob/e2c5af3d7b14f5a92b1ece2bc972ebd853d79c6c/src/materials/nodes/SpriteNodeMaterial.js#L72